### DISCUSSION TITLE
Make single quotes for string property optional

### DISCUSSION THREAD

**Discussion by Janry** [serilog-contrib/serilog-sinks-grafana-loki#130] [Ideas]
Hi. 
`
    I have problem with single quotes. In code you replace all "\"" to "'". Can you please make this replace optional. I need just empty string for this. Because for example enum.SomeValue convert to "SomeValue", but string "SomeValue" to "'SomeValue'" and it's not looks like good.
    I solved my problem with ILogEventEnricher and LogEventPropertyValue inherit my log event property from ScalarValue and override Render. But with parameter in appsettings.json will be better. 

Thank you.

**Comment by mishamyte** [comment]
Hi @Janry,

Will look into that. We do some replacing like [that](https://github.com/serilog-contrib/serilog-sinks-grafana-loki/blob/master/src/Serilog.Sinks.Grafana.Loki/LokiBatchFormatter.cs#L200-L201) to fix a payload.

I will look into your case

**Reply by mishamyte** [reply]
Also @Janry, 

Could you provide a simple repro, cause I've tested on a simple example:
![image](https://user-images.githubusercontent.com/5649656/191693001-3a565571-749d-4691-bf66-706758da3a5d.png)

And seems like here are no extra quotes:
![image](https://user-images.githubusercontent.com/5649656/191693140-dd209e9d-96b0-469f-9b4b-d1bdb852c538.png)

Loki's query is `{app="web_app"} | json`

**Reply by Janry** [reply]
Hi, @mishamyte. Thank you for your reply.

When you add global property in Lables in appsettings.json it looks good without escape single quotes.
When you use Enrich.WithProperty with runtime values it has single quotes escape.

![image](https://user-images.githubusercontent.com/1222756/191821134-49feaec8-abb5-4930-b3b5-97eae1b18139.png)

![image](https://user-images.githubusercontent.com/1222756/191822441-cc457b80-312a-41b0-a920-540c7a57f030.png)

**Reply by mishamyte** [reply]
What version of package do you use? And also Loki + Grafana version would be nice to know.

So I just tested for `Enrich.WithProperty` (if I understand correctly, `WithUnquoutedProperty` is something yours).

So I modified the console sample from this repo. Added `cluster` property.

```csharp
Log.Logger = new LoggerConfiguration()
            .MinimumLevel.Debug()
            .Enrich.WithThreadId()
            .Enrich.WithProperty("meaning_of_life", 42)
            .Enrich.WithProperty("cluster", "local")
            .WriteTo.Console(outputTemplate: OutputTemplate)
            .WriteTo.GrafanaLoki(
                "http://localhost:3100",
                new List<LokiLabel> { new() { Key = "app", Value = "console" } },
                credentials: null)
            .CreateLogger();
```

In this case I got the next results:
![image](https://user-images.githubusercontent.com/5649656/191824091-8280cc73-b5fd-4723-9a67-0eb3c64aa667.png)

My Loki query is `{app="console"} | json`

So you could see that there are no single quotes here. Detected fileds are in double quoutes and when it is parsed to labels by a `| json` parser, there are no quotes remaining.

For this test I used `Serilog.Sinks.Grafana.Loki` v8.0.0, `Grafana` v9.1.6 and `Loki` v2.6.1

**Reply by mishamyte** [reply]
Hi @Janry,

Any suggestions for this case from your side?
